eox_tagging/test/test_viewset.py

```diff
@@ -93,9 +93,13 @@ def test_filter_by_tag_key(self):
 
     def test_filter_by_username(self):
         """Used to test getting a tag given its target."""
-        URL = "/api/v1/tags/?username={user}".format(user="user_test")
+        USERNAME = "user_test"
+        URL = "/api/v1/tags/?username={user}".format(user=USERNAME)
```

Comment:
reverse, https://docs.djangoproject.com/en/1.11/topics/testing/tools/#django.test.Client.get

Reply:
I'll rebase this PR and fix it